app/models/VerifiedEmailAddress.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import play.api.libs.json.{Json, OFormat}
+
+import java.time.Instant
+
+case class VerifiedEmailAddress(address: String, timestamp: Instant)
```

Comment:
Not super important because we do not use `timestamp` atm and (probably) we'll never have problems with time zones, but if you want to stay on the safe side I would change the `Instant` type to `ZonedDateTime`.
You can read on [Stackoverflow](https://stackoverflow.com/questions/52690684/instant-vs-zoneddatetime) reg. the differences between the 2 types.

Reply:
Yes good spot. Swapped out.